<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI::InterfaceSettingsWidget</class>
 <widget class="QWidget" name="UI::InterfaceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="languageGroupBox">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QVBoxLayout" name="languagesLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QComboBox" name="languageSelect"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="playlistGroupBox">
     <property name="title">
      <string>Playlist</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelPlaylistCachedFiles">
        <property name="toolTip">
         <string>Specifies how many opened files can be cached per playlist</string>
        </property>
        <property name="text">
         <string>Cache size, files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="playlistCachedFiles">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPlaylistCachedMemory">
        <property name="toolTip">
         <string>Specifies per playlist cache memory usage limit</string>
        </property>
        <property name="text">
         <string>Cache size, MiB</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="playlistCacheLimit">
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="playlistStoreAllProperties">
        <property name="toolTip">
         <string>Store all module attributes and properties instead of only custom</string>
        </property>
        <property name="text">
         <string>Store full modules' attributes in saved playlists</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="appGroupBox">
     <property name="title">
      <string>Application</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="updateCheckLabel">
        <property name="text">
         <string>Check updates</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="updateCheckPeriod">
        <item>
         <property name="text">
          <string>Never</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Once a day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Once a week</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cmdlineTargetLabel">
        <property name="toolTip">
         <string>Target for items passed via commandline</string>
        </property>
        <property name="text">
         <string>Command line destination</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmdlineTarget">
        <item>
         <property name="text">
          <string>New playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Active playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Visible playlist</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="appSingleInstance">
        <property name="toolTip">
         <string>Changes applied after application restart</string>
        </property>
        <property name="text">
         <string>Single instance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
